src/status_im/contexts/wallet/home/view.cljs

```diff
@@ -26,6 +27,28 @@
       :on-press            #(rf/dispatch [:navigate-to :screen/wallet.add-address-to-watch])
       :add-divider?        true}]]])
 
+(defn share-qr-code
```

Comment:
imo this should not live here, it's more related to share then it is to wallet, no?
Can't we make it as part of the dispatch from the share screens and remove the use-effect below too?